csrc/fusion_segmenter.cpp

```diff
@@ -1934,10 +1934,13 @@ std::unique_ptr<SegmentedFusion> SegmentCandidateFinder::segment(
       return SegmentedFusion::fromCompleteFusion(
           std::move(fusion), maybe_complete_fusion_heuristic.value(), *inputs);
     }
+  } else {
+    scheduler_debug_utils::canScheduleMessage(
```

Comment:
Is it intended to remove the debug message on `segment hint`?

Reply:
> (2) If segmented with hint:

Looking at the commit message, I think we are still expecting to print the debug message?

Reply:
the debug message on `segment hint` is being moved into the `else` branch. So we only dump this msg when there is a `segment hint`.
The current main branch, dumps `Has segment hints...` when segmented, even without an actual `segment hint.`